Classes/Controller/ModuleController.php

```diff
@@ -468,6 +469,44 @@ public function fileReferenceCreateAction($file, $element): ResponseInterface
         return $this->htmlResponse();
     }
 
+    public function getPasswordHash(String $password, String $mode) : String {
+        $hashInstance = GeneralUtility::makeInstance(PasswordHashFactory::class)
```

Comment:
as far as i am aware of, we should consider to move to dependency injection for factories and other services as this has become default handling in TYPO3 v11. but this could be handled in a different patch ..